pkg/virt-config/config-map.go

```diff
@@ -463,6 +480,25 @@ func setConfigFromKubeVirt(config *v1.KubeVirtConfiguration, kv *v1.KubeVirt) er
 	return nil
 }
 
+// getCPUArchSpecificDefault get arch specific default config
+func getCPUArchSpecificDefault(cpuArch string) (string, string, []string) {
+	if cpuArch == "" {
```

Comment:
I think that this is not needed. I would rather consider failing if cpuArch is not set at this stage.

Reply:
> I think that this is not needed. I would rather consider failing if cpuArch is not set at this stage.

I will amend it in the following PR.